docs/op-stack/src/.vuepress/styles/index.styl

```diff
@@ -19,13 +19,13 @@ aside.sidebar {
 p.sidebar-heading {
   color: #323A43 !important;
   font-family: 'Open Sans', sans-serif;
-  font-weight: 600 !important;
```

Comment:
Should be OK to keep the current file as-is here

Reply:
Resolved by using the current index.styl instead of the one from devdocs.